indextests.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+// Testing code as writing, little by little
+require_once 'RecipesDao.php';
+require_once 'Recipes.php';
+
+$recipeDao = new RecipeDao();
```

Comment:
You should remove any files used for testing stuff like this

Reply:
Removed